### PR TITLE
Update CGAL to version 4.14

### LINKED ISSUE #5888: CGAL Library update
Please update the CGAL library to version 4.14. 

cgal  4.13-2           The Computational Geometry Algorithms Library (CGAL) is a C++ library that aim

https://www.cgal.org

Many thanks in advance.

### PR DESCRIPTION
**Pull request by lrineau**
<s>Do not merge yet!</s>

This pull-request is a test to update the CGAL port to version 4.14-beta2. That CGAL version is currently a beta version. I will update the port once the official release it out.

Fix #5888.

### CODE REVIEW

File: ports/cgal/portfile.cmake [hunk 1]
```diff
@@ -3,11 +3,16 @@ include(vcpkg_common_functions)
...
     OUT_SOURCE_PATH SOURCE_PATH
     REPO CGAL/cgal
-    REF releases/CGAL-4.13
-    SHA512 3a12d7f567487c282928a162a47737c41c22258556ca0083b9cf492fc8f0a7c334b491b14dbfd6a62e71feeeb1b4995769c13a604e0882548f21c41b996d4eaf
+    REF releases/CGAL-4.14-beta2
+    SHA512 edbc4d656ba3795a0d6ee4ec9cabc686bdd00c040475145923256010737b08c5287661c1099cb78538e9e23b2f0e50b286cd3a68e48651e559c131284457c263
     HEAD_REF master
 )
 
+vcpkg_apply_patches(
```
Comment (Rastaban) [inline ports/cgal/portfile.cmake]: This can be done with the PATCHES argument to vcpkg_from_github

**Comment by Rastaban** [comment]
Looks good so far and passes all the tests.  When do you expect the release to happen?

**Comment by lrineau** [comment]
[**@Rastaban**](https://github.com/Rastaban) commented on [Mar 19, 2019, 9:25 PM GMT+1](https://github.com/Microsoft/vcpkg/pull/5734#issuecomment-474567579 "2019-03-19T20:25:14Z - Replied by Github Reply Comments"):
> Looks good so far and passes all the tests.  When do you expect the release to happen?

Hopefully next week. Unfortunately, we discovered a bug in our CMake scripts in CGAL-4.14-beta1, and then this one in CGAL-4.14-beta2. The difficulty with bugs in CMake scripts is that our daily test suite cannot test all the possible usages (we try our best). That is why I wanted to test the beta releases with the downstream package managers (like vcpkg and brew). I am wondering if we should publish a third beta release.

Cc: @maxGimeno

**Comment by lrineau** [comment]
@Rastaban The final (non-beta) version of CGAL-4.14 has been published, and we have updated the PR. It is now ready to be evaluated and eventually merged.

**Pull request merged**